src/Tickets/Commerce/Order_Modifiers/Checkout/Gateway/PayPal/Fees.php

```diff
@@ -10,7 +10,7 @@
  * @package TEC\Tickets\Commerce\Order_Modifiers\Checkout\Gateway\Paypal
  */
 
-namespace TEC\Tickets\Commerce\Order_Modifiers\Checkout\Gateway\Paypal;
+namespace TEC\Tickets\Commerce\Order_Modifiers\Checkout\Gateway\PayPal;
```

Comment:
`capital_P_dangit`? 😆

Reply:
I know - i was looking like crazy to understand why my tests were failing...